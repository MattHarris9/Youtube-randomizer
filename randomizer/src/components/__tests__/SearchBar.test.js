import React from 'react';
import renderer from 'react-test-renderer';
import SearchBar from '../playlist/SearchBar';

describe('Search Bar', () => {
  test('renders emptysearch bar correctly', () => {
    const tree = renderer.create(<SearchBar />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
