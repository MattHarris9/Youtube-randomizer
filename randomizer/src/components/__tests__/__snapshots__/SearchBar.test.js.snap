// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search Bar renders emptysearch bar correctly 1`] = `
<div
  className="MuiGrid-root WithStyles(ForwardRef(Grid))-root-1 MuiGrid-container MuiGrid-spacing-xs-1 MuiGrid-align-items-xs-center"
>
  <div
    className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-10 MuiGrid-grid-md-11 MuiGrid-grid-lg-11"
    s={11}
  >
    <div
      className="MuiFormControl-root MuiTextField-root WithStyles(ForwardRef(TextField))-root-105 MuiFormControl-marginNormal"
    >
      <div
        className="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-colorSecondary MuiInput-colorSecondary MuiInputBase-formControl MuiInput-formControl"
        onClick={[Function]}
      >
        <input
          aria-invalid={false}
          className="MuiInputBase-input MuiInput-input"
          disabled={false}
          id="search"
          onAnimationStart={[Function]}
          onBlur={[Function]}
          onChange={[Function]}
          onFocus={[Function]}
          placeholder="Enter a youtube playlist id or playlist URL"
          required={false}
          type="text"
        />
      </div>
    </div>
  </div>
  <div
    className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-2 MuiGrid-grid-md-1 MuiGrid-grid-lg-1"
    s={1}
  >
    <button
      className="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary"
      disabled={false}
      onBlur={[Function]}
      onDragLeave={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      style={
        Object {
          "width": "100%",
        }
      }
      tabIndex={0}
      type="button"
    >
      <span
        className="MuiButton-label"
      >
        Go
      </span>
    </button>
  </div>
</div>
`;
